Source/Model/Conversation/ZMConversation+Mute.swift

```diff
@@ -140,6 +140,12 @@ extension ZMConversationMessage {
             return false
         }
         
+        // We assume that all composite messages are alarming messages
+        let compositeMessage = self as? ConversationCompositeMessage
+        guard compositeMessage?.compositeMessageData == nil else {
```

Comment:
`compositeMessage` is not used below so we can:

```suggestion
        guard (self as? ConversationCompositeMessage)?.compositeMessageData == nil else {
```